module/chunks/chunkVerifier_test.go

```diff
@@ -69,10 +76,11 @@ func (s *ChunkVerifierTestSuite) TestMissingRegisterTouchForUpdate() {
 
 // TestMissingRegisterTouchForRead tests verification given a chunkdatapack missing a register touch (read)
 func (s *ChunkVerifierTestSuite) TestMissingRegisterTouchForRead() {
+	s.T().Skip("Check new partial ledger for missing keys")
```

Comment:
I thought we fixed this ?

Reply:
That's a great spot. Not sure if those tests are applicable, as they check for missing register touches, and we don't have those in chunks anymore